neqo-transport/src/lib.rs

```diff
@@ -75,6 +78,8 @@ pub use self::{
     version::Version,
 };
 
+pub type IndexMap<K, V> = indexmap::IndexMap<K, V, BuildHasherDefault<FxHasher>>;
```

Comment:
Why is this more performant? Can you add a comment?

Reply:
It's probably nor, see above as to why.